openmetadata-ui/src/main/resources/ui/src/components/common/OwnerItem/OwnerItem.tsx

```diff
@@ -38,24 +36,27 @@ export const OwnerItem: React.FC<OwnerItemProps> = ({
   ownerDisplayName,
   avatarSize = 32,
 }) => {
-  const { t } = useTranslation();
   const displayName = getEntityName(owner);
   const ownerPath = getOwnerPath(owner);
 
   const inheritedIcon = owner?.inherited ? (
-    <Tooltip
-      title={t('label.inherited-entity', {
-        entity: t('label.owner-plural'),
-      })}>
-      <InheritIcon className="inherit-icon cursor-pointer" width={8} />
-    </Tooltip>
+    <InheritIcon className="inherit-icon cursor-pointer" width={8} />
   ) : null;
 
   return (
     <div
-      className="d-inline-flex items-center owner-avatar-container gap-1"
+      className={classNames('owner-avatar-container', {
+        'is-compact-view': isCompactView,
+      })}
       style={{
-        marginLeft: index === 0 || isCompactView ? 0 : '-4px',
+        zIndex: !isCompactView ? index + 1 : undefined, // Lower index items will be underneath
+        position: !isCompactView ? 'relative' : undefined,
+        ...(isCompactView && {
+          display: 'flex',
```

Comment:
Did we check this? https://4x.ant.design/components/avatar/#components-avatar-demo-group

Reply:
Yes, had checked this initially. But this Owner Label component was not using this from start itself . Not sure if there were limitations in it. 
Also now,with the current new Avatar design which has reverse overlapping and different style for the max tags button , i think its difficult to customize the Ant design's Avatar Group. Not able to find any prop support for the same.